### ISSUE TITLE
email.notif_template_html and  email.notif_template_text are missing from sample config

### ISSUE THREAD

**Issue by richvdh** [matrix-org/synapse#6960]
If you set `email.enable_notifs: True`, synapse will refuse to start unless you also set `email.notif_template_html` and `email.notif_template_text`, but these are not mentioned in the sample config

**Comment by richvdh** [comment]
stacktrace:

```
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/opt/venvs/matrix-synapse/lib/python3.6/site-packages/synapse/app/homeserver.py", line 639, in <module>
    main()
  File "/opt/venvs/matrix-synapse/lib/python3.6/site-packages/synapse/app/homeserver.py", line 634, in main
    hs = setup(sys.argv[1:])
  File "/opt/venvs/matrix-synapse/lib/python3.6/site-packages/synapse/app/homeserver.py", line 319, in setup
    "Synapse Homeserver", config_options
  File "/opt/venvs/matrix-synapse/lib/python3.6/site-packages/synapse/config/_base.py", line 617, in load_or_generate_config
    config_dict, config_dir_path=config_dir_path, data_dir_path=data_dir_path
  File "/opt/venvs/matrix-synapse/lib/python3.6/site-packages/synapse/config/_base.py", line 639, in parse_config_dict
    data_dir_path=data_dir_path,
  File "/opt/venvs/matrix-synapse/lib/python3.6/site-packages/synapse/config/_base.py", line 254, in invoke_all
    res[name] = getattr(config, func_name)(*args, **kwargs)
  File "/opt/venvs/matrix-synapse/lib/python3.6/site-packages/synapse/config/emailconfig.py", line 264, in read_config
    % (", ".join(["email." + k for k in missing]),)
RuntimeError: email.enable_notifs is True but required keys are missing: email.notif_template_html, email.notif_template_text
```

**Comment by richvdh** [comment]
Introduced in #6685

**Comment by richvdh** [comment]
fixed by #6962
